src/java/picard/analysis/CompareMetrics.java

```diff
@@ -17,13 +17,22 @@
  * Compare two metrics files.
  */
 @CommandLineProgramProperties(
-        usage = CompareMetrics.USAGE,
-        usageShort = CompareMetrics.USAGE,
+        usage = CompareMetrics.USAGE_SUMMARY + CompareMetrics.USAGE_DETAIL,
+        usageShort = CompareMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CompareMetrics extends CommandLineProgram {
-
-    static final String USAGE = "Compare two metrics files";
+    static final String USAGE_SUMMARY =  "Compares two metrics files";
+    static final String USAGE_DETAIL = "Compares the headers of the two input metrics files that have the same structure," +
+            " but come from different underlying metric classes.  Outputs can be either equal" +
```

Comment:
Is the result given in the output log (which we would express as "written to stdout"), as opposed to being written to file?